index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <link rel="stylesheet" href="./css/style.css">
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto&display=swap" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Lato:wght@900&family=Poppins:wght@100;300&display=swap"
+    rel="stylesheet">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Conference</title>
+</head>
+
+<body>
+  <section class="navbar">
+    <div class="menu-icon">
+      <hr class="m-menu">
+      <hr class="m-menu">
+      <hr class="m-menu">
+    </div>
+    <ul class="menu-ul">
+      <div class="head-line">
+        <img alt="facebook" class="head-img" src="./Images/facebook.png">
+        <img alt="facebook" class="head-img" src="./Images/tweeter.png">
+        <li class="head-link"> English </li>
+        <li class="head-link"> Mypage</li>
+        <li class="head-link">Logout</li>
+      </div>
+      <img class="close" alt="close" src="./Images/close.png">
+      <img alt="logo" class="header-logo" src="./Images/logo.png">
+      <li class="links link1"><a class="menu-links" href="#home">Home</a></li>
+      <li class="links link2"><a class="menu-links" href="./about.html">About</a></li>
+      <li class="links link3"><a class="menu-links" href="#program">program</a></li>
+      <li class="links link3"><a class="menu-links" href="#program">Speakers</a></li>
+    </ul>
```

Comment:
- [OPTIONAL] According to the project requirement, The pages should look almost identical to the original design, outside the content differences. what if you add this 👇🏻 missing menu option to your navbar-links 😄 without forgetting to add a border: with style color and use the same color as from the original template 💯 

![nav1](https://user-images.githubusercontent.com/79058364/168643439-051c8a69-818b-47fe-af98-bde68f57e2dc.png)